src/ophyd_async/epics/signal/__init__.py

```diff
@@ -1,4 +1,8 @@
-from .signal import (
+from ._common import LimitPair, Limits, get_supported_values
+from ._epics_transport import EpicsTransport
+from ._p4p import PvaSignalBackend
+from ._signal import (
+    _make_backend,
```

Comment:
Should: make this public and give it a different name
```suggestion
    epics_signal_backend,
```

Reply:
I have changed it as requested but this feels like it should just be a private function, its only used in signal to make the required signal backends.

Reply:
I would even argue that it should be omited from the init and imported just for the tests with a noqa.

Reply:
Refactored the test to not use it so it has been renamed but it now private.